server/core/src/irods_signal.cpp

```diff
@@ -26,14 +29,12 @@ void stacktrace_signal_handler(int _signal)
 
     // "_POSIX_PATH_MAX" is guaranteed to be defined as 256.
     // This value represents the minimum path length supported by POSIX operating systems.
-    // Given that we're dealing with a relative path, a buffer of this size is acceptable here.
-    char file_path_unprocessed[_POSIX_PATH_MAX] = "../../";
-    std::strcat(file_path_unprocessed, IRODS_DEFAULT_PATH_HOMEDIR);
-    std::strcat(file_path_unprocessed, "/stacktraces/");
-
+    char file_path_unprocessed[_POSIX_PATH_MAX];
```

Comment:
Since we're not making assumptions about the working directory (and therefore not using relative paths), I don't know if `_POSIX_PATH_MAX` makes sense to use any more.

Reply:
We have to be careful in this function since it must be async-signal-safe.

`_POSIX_PATH_MAX` was chosen because it is consistent across POSIX implementations. If we don't use that, what would you recommend the size of the buffer be?

Reply:
We have to be careful in this function since it must be async-signal-safe.

`_POSIX_PATH_MAX` was chosen because it is consistent across POSIX implementations. If we don't use that, what would you recommend the size of the buffer be?